resources/views/adm/mship/feedback/settings.blade.php

```diff
@@ -105,6 +81,13 @@
             $("#feedback-form-questions").on("click", ".question-delete-button", function () {
                 $(this).closest('.question-item').remove();
             });
+
+            // Question accordion control
+            $("#feedback-form-questions").on( 'click', '.question-settings-control', function () {
```

Comment:
This is only present on `settings.blade.php` - needs copying over for new forms.